### PR TITLE
Organisation type field

### PR DESCRIPTION
**Pull request by davidrapson**
image

### CODE REVIEW

File: controllers/apply/get-advice/form.js [hunk 1]
```diff
@@ -412,6 +412,137 @@ module.exports = function({ locale = 'en', data = {} } = {}) {
...
+    function fieldOrganisationType() {
+        return new RadioField({
+            name: 'organisationType',
+            label: localise({
+                en: 'What type of organisation are you?',
+                cy: 'Pa fath o sefydliad ydych chi?'
+            }),
+            options: [
+                {
+                    value: 'unregistered-vco',
```
Comment (davidrapson) [inline controllers/apply/get-advice/form.js]: I'm deliberately not using any constants here or sharing the code from awards for all a) we don't need to depend on organisation type for sub-questions here and b) the actual list may diverge from the simple product (e.g. different options based on country, and some different valid types) so doesn't make sense to share anything.

**Comment by davidrapson** [comment]
[Ask for organisation type](https://trello.com/c/dPYdCUfw/827-ask-for-organisation-type)

File: controllers/apply/lib/field-types/field.js [hunk 2]
```diff
@@ -19,34 +17,77 @@ class Field {
...
+     */
+    defaultLabel() {
+        return null;
+    }
+
+    /**
+     * Allow sub-classes to provide a default schema
+     * also allows it to be accessed by the instance
+     */
+    defaultSchema() {
```
Comment (davidrapson) [inline controllers/apply/lib/field-types/field.js]: Added these `default*` methods to make it easier for the sub-types to define their own without having to duplicate as much boilerplate in the constructor. See `phone.js` for a good example of this.

File: controllers/apply/lib/field-types/field.js [hunk 3]
```diff
@@ -19,34 +17,73 @@ class Field {
             throw new Error('Must provide name');
         }
 
-        this.label = props.label;
+        const label = props.label ? props.label : this.defaultLabel();
+        if (label) {
+            this.label = label;
+        } else {
+            // throw new Error('Must provide label');
```
Comment (mattandrews) [inline controllers/apply/lib/field-types/field.js]: Should this be here/commented?

**Pull request merged**